alarms/utils.py

```diff
@@ -170,4 +172,23 @@ def get_pos_by_name(location_name):
 	
 def time_fix(t):
 	diff = datetime.utcnow() - datetime.utcfromtimestamp(time.mktime(datetime.utcnow().timetuple()))
-	return t+diff
\ No newline at end of file
+	return t+diff
+	
+#Return true if x,y points are inside polygon
+def point_in_poly(x,y):
```

Comment:
Can we have some more commenting explaining the method used to determine if the point is in the geofence?